lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -195,7 +195,7 @@ private static function get_inner_blocks_from_navigation_post( $attributes ) {
 
 			// 'parse_blocks' includes a null block with '\n\n' as the content when
 			// it encounters whitespace. This code strips it.
-			$compacted_blocks = block_core_navigation_filter_out_empty_blocks( $parsed_blocks );
+			$compacted_blocks = gutenberg_block_core_navigation_filter_out_empty_blocks( $parsed_blocks );
```

Comment:
@scruffian I'm seeing that [these references have reverted to `block_` prefix in Gutenberg `trunk`](https://github.com/WordPress/gutenberg/blob/e0005a813686157e2f25f6d4b28cc72ebadbfaae/packages/block-library/src/navigation/index.php#L193). Can you verify if that was intentional so that we can avoid any regressions here 🙏

Reply:
My understanding is that this is now ok because functions being called are now within block-library and indeed within the same _file_.

Reply:
Yeah that's my understanding too, so I think this is fine.

Reply:
Yes that's ok! What we _shouldn't_ have is `gutenberg_` prefixed functions in block-library.

Reply:
That's a good point. I think that's what must have drawn my attention to this change.

@scruffian why are we now using `gutenberg_` prefix? Should this be revised as @tellthemachines indicates?

Reply:
We don't have the `gutenberg_` prefix in the navigation block except where its added by the webpack script.